recipes/datatable/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "datatable" %}
+{% set version = "0.11.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datatable-{{ version }}.tar.gz
+  sha256: 19c602711e00f72e9ae296d8fa742d46da037c2d3a2d254bdf68f817a8da76bb
+
+build:
+  number: 0
+  skip: true  # [py<35 or win]
+
+# numpy and pandas are _optional_ dependencies of dataframe
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+  run:
+    - python
+    # fix overlinking error
+    - libgcc   # [linux]
+    - libcxx   # [osx]
```

Comment:
This definitely isn't needed to make the CI pass. If it fails ping me and I'll take a look:
```suggestion
```

Reply:
@chrisburr I think the following shows that it is required in order to make the CI pass:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=248328&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0

Reply:
It was still `noarch: python` at the time despite containing platform specific binaries

Reply:
Cool, I've committed that change and it looks like CI passes now.